cmake/HPX_PerformCxxFeatureTests.cmake

```diff
@@ -74,6 +75,8 @@ function(hpx_perform_cxx_feature_tests)
   # C++20 feature tests
   hpx_check_for_cxx20_coroutines(DEFINITIONS HPX_HAVE_CXX20_COROUTINES)
 
+  hpx_check_for_cxx20_disable_sized_sentinel_for(DEFINITIONS HPX_HAVE_CXX20_DISABLE_SIZED_SENTINEL_FOR)
```

Comment:
Same here, please rename to `HPX_HAVE_CXX20_STD_DISABLE_SIZED_SENTINEL_FOR`. Also cmake-format this file.